images/win/post-generation/Dotnet.ps1

```diff
@@ -0,0 +1,9 @@
+$latestPath = [System.Environment]::GetEnvironmentVariable('PATH', [System.EnvironmentVariableTarget]::Machine)
+$dotnetPath = "$env:USERPROFILE\.dotnet\tools"
+
+if (-not $latestPath.Contains($dotnetPath))
+{
+    $latestPath = "$dotnetPath;$latestPath"
+}
+
+[System.Environment]::SetEnvironmentVariable('PATH', $latestPath, [System.EnvironmentVariableTarget]::Machine)
```

Comment:
Does it make sense to move this line inside `if` block?

Reply:
Yep, agree on that. Moved inside `if` block.